src/rock/models/finances/models/TransactionJobs.js

```diff
@@ -233,12 +235,8 @@ export default class TransactionJobs extends Rock {
       return data;
     }
 
-    if (!Transaction.SourceTypeValueId && SourceTypeValue.Url) {
-      Transaction.SourceTypeValueId = await DefinedValue.findOne({
-        where: { Value: SourceTypeValue.Url, DefinedTypeId: 12 },
-      })
-        .then(x => x && x.Id || 10);
-    }
+    // 884 source type: (native.ns.cc), default 798: (my.ns.cc)
+    Transaction.SourceTypeValueId = platform === "Native" ? 884 : 798;
```

Comment:
@JakeDawkins Why hard code the numbers? Are they defined values in Rock? Should we be pulling them from there?